worker-core/readme.md

```diff
@@ -580,7 +683,6 @@ the current input queue. Default is True.
       return resultQueue;
     }
 
-
```

Comment:
Don't drop this blank line.  There are two blank lines between all the other methods.